### PR TITLE
Brumik link top templates to job expoler

### PR DESCRIPTION
**Pull request by brumik**
#183 

Adds a link to the top templates modal for the job-explorer page. Build on top of: `jo_job-explorer-list-view` branch.

Additionally fixes a console error: when `clusters` page was loaded it automatically loaded the modal window as well because of a wrong if statement and throw a `Modal needs a title or ...` error in the console. 

**Before**:
Screenshot from 2020-07-22 16-57-12

**After**:
Screenshot from 2020-07-24 15-14-39

**After clicking on `View all jobs`:**
Screenshot from 2020-07-23 13-52-36

### CODE REVIEW

File: src/Components/TemplatesList.js [hunk 1]
```diff
@@ -88,7 +91,14 @@ const formatTopFailedTask = data => {
     return `Unavailable`;
 };
 
-const TemplatesList = ({ templates, isLoading, queryParams }) => {
+const borderedButtonStyle = {
```
Comment (kialam) [inline src/Components/TemplatesList.js:94]: Should use the PF `label` component for this: https://www.patternfly.org/v4/documentation/react/components/label#outline

**Comment by kialam** [comment]
New elements should be vertically aligned as per the mockups
Screen Shot 2020-07-23 at 12 49 13 PM

File: src/Components/TemplatesList.js (hunk 1 above)
Comment (brumik) [reply src/Components/TemplatesList.js:94]: ah, I was searching for this, but could not find anything... never thought of label, thanks

**Comment by Ladas** [comment]
@brumik so to get the same counts:

Pass the main filter values of clusters page:
- cluster (cluster_id)
- date_range (pass exactly the same start_date and end_date)

Then other filters are:
- status (none selected, we filter all jobs)
- job_type: [Playbook run]

(count should fit, the list of jobs with `make data` will be probably different, since many jobs have the same `created` timestamp, the order gets kind of random. In prod, the probability that 2 jobs will have a same created down to microsecond is ~0)

**Comment by brumik** [comment]
The numbers are the same now on both pages after clicking the link. @kialam @AlexSCorey

**Pull request merged**